crates/bevy_pbr/src/render/light.rs

```diff
@@ -616,6 +634,14 @@ pub fn prepare_lights(
         .filter(|light| light.1.spot_light_angles.is_none())
         .count();
 
+    let point_light_volumetric_enabled_count = point_lights
+        .iter()
+        .filter(|(_, light, _)| {
+            (light.volumetric || light.shadows_enabled) && light.spot_light_angles.is_none()
```

Comment:
Why do you need the shadows_enabled here?

Reply:
To ensure that the volumetric spot light is activated correctly after the non-volumetric, shadow enabled point light. A better approach would be to subtract `point_light_count ` instead. Let me optimize the code, thank you.